### PR TITLE
Add GitHubPRComment Task

### PR DESCRIPTION
**Pull request by ParadoxARG**
Move GitHubPRComment Task from [Botbuilder-dotnet](https://github.com/southworkscom/botbuilder-dotnet/tree/feature/APT/github-pr-comment/add-multiple-files)

### CODE REVIEW

**Review by gasper-az — APPROVED** [review]
Solve the conflicts with the file `Tasks/ApiCompat/vss-extension.json` and :shipit:

File: Tasks/GitHubPRComment/githubComment/Utilities.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export class AzureDevOpsVariables {
```
Comment (VictorGrycuk) [inline Tasks/GitHubPRComment/githubComment/Utilities.ts]: Is this file being used? I couldn't find a reference to it.
If it is being used, I would recommend changing it like this:
```javascript
 public static readonly pullRequestNumber: string = getVariable('System.PullRequest.PullRequestNumber');
```
Or simply creating a class to handle all the task's inputs and variables

**Review by VictorGrycuk — CHANGES_REQUESTED** [review]
There is a class that seems to not be being used.
Also, I would recommend removing the `.js` files.

**Review by VictorGrycuk — APPROVED** [review]

**Comment by VictorGrycuk** [comment]
🚢 it!

**Pull request merged**